### DISCUSSION TITLE
Change Amount Shown in Dropdown

### DISCUSSION THREAD

**Discussion by dbltoe** [marco-pm/zencart_instantsearchplus#9] [Q&A] [answered]
REALLY like this version but, I'd like to have the option to change the number of items shown in the dropdown.

Can you point me in the right direction?

**Comment by dbltoe** [comment, marked as answer]
Found it in line 16 of includes/classes/ajax/zcAjaxInstantSearch.php

**Reply by marco-pm** [reply]
great!

Soon I'll add the features that are still missing and the configuration options in the admin.
